.github/workflows/pr.yml

```diff
@@ -3,95 +3,274 @@ name: GitHub Actions Pull Request
 on:
   pull_request:
 
+# Note: see BIN_PATH for how to set an env variable that depends on
+# other env variables.
 env:
-  LOG_DIR: ~/log
-  BIN_DIR: ~/bin
   LEIN_ROOT: 1
+  LEIN_VERSION: 2.9.4
   CTIA_MAJOR_VERSION: 1
-  COMPOSE_URI: https://github.com/docker/compose/releases/download/1.9.0
-  COMPOSE_BIN: ~/bin/docker-compose-1-9-0
-  COMPOSE_LOG: ~/log/docker-compose.log
-  CTIA_STORE_ES_DEFAULT_HOST: 127.0.0.1
-  CTIA_STORE_ES_DEFAULT_INDEXNAME: elasticsearch
-  CTIA_STORE_ES_DEFAULT_CLUSTERNAME: elasticsearch
-  CTIA_HOOK_ES_HOST: 127.0.0.1
-  CTIA_HOOK_ES_PORT: 9200
-  CTIA_HOOK_ES_INDEXNAME: elasticsearch
-  CTIA_HOOK_REDIS_URI: redis://127.0.0.1:6379
   # number of test-matrix builds
-  # (9 jobs for test-matrix + 1 for shellcheck) + 1 for fan-in
+  # 1 setup + 10 test jobs + 1 for fan-in
   # allows 2 commits to build simultaneously, since one build
   # will take at most 10 builds at any one time
-  CTIA_NSPLITS: 9
+  CTIA_NSPLITS: 10
```

Comment:
I would keep the same number of splits to compare the build time with previous solution.